NeoMathEngine/src/GPU/Vulkan/VulkanMathEngineDnnPoolings.cpp

```diff
@@ -89,10 +91,9 @@ C3dMaxPoolingDesc* CVulkanMathEngine::Init3dMaxPooling( const CBlobDesc& source,
 }
 
 void CVulkanMathEngine::Blob3dMaxPooling( const C3dMaxPoolingDesc& poolingDesc, const CFloatHandle& sourceData,
-	const CIntHandle* maxIndicesData, const CFloatHandle& resultData )
+	const CIntHandle*, const CFloatHandle& resultData )
 {
 	ASSERT_EXPR( sourceData.GetMathEngine() == this );
-	ASSERT_EXPR( maxIndicesData == 0 );
```

Comment:
Why it was removed? It's supposed to prevent unexpected behavior (in case user expects `maxIndicesData` to be calculated, but `CVulkanMathEngine` doesn't support this scenario)